backend/endpoints/portfolio/route.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+from ninja import Router, Schema
+from lib.polygon_api import PolygonAPI
+
+from endpoints.auth import AuthBearer, AuthenticatedRequest
+from models.models import HoldingModel, UserModel
+
+router = Router(auth=AuthBearer())
+
+
+class StatsResponseSchema(Schema):
+    current_value: float
+
+
+@router.get("/stats", response=StatsResponseSchema)
+def stats(request: AuthenticatedRequest) -> StatsResponseSchema:
+    user = UserModel.objects.get(username=request.auth)
+    holdings = HoldingModel.objects.filter(user=user)
+
+    api = PolygonAPI()
+
+    current_value = sum([api.recent_price(holding.stock.ticker).low * float(holding.units)
+                        for holding in holdings])
+
+    return StatsResponseSchema(current_value=current_value)
```

Comment:
Would probably like today's gain/loss and overall gain/loss to be returned as well, since it seems easier to be done on the backend?

Reply:
This is also trickier than it seems. What would be the gain/loss to return if the markets are closed today?

Reply:
To be fair what I was using in the watchlist is to subtract the open price of the day from the close price of the same day - which is something that is strictly speaking incorrect (ideally, should be close price of the last day subtracted from price at the moment) but is easier to achieve. So in this case if the markets are closed today, what actually gets returned depends on which day's data the call to `recent_price` returns - if it's for the next incoming market day then we actually get a g/l of 0.

Reply:
Okay I kinda get what you mean. Will set this as an issue for someone else to pick up :)